File: Code/cpp/selection_sort.cpp
#include<iostream>
using namespace std;
int main()
{
    int length, temp, i, j;

    cout << "Enter Length: ";
    cin >> length;
    int arr[length];
    for (i = 0; i < length; i++)
    {
        cout << "Enter Number: ";
        cin >> arr[i];
    }

    for (i = 0; i <  length; i++)
    {
        for (j = i + 1; j < length; j++)
        {
            if (arr[i] > arr[j])
            {
                temp = arr[j];
                arr[j] = arr[i];
                arr[i] = temp;
            }
        }
    }
    for (i = 0; i < length; i++)
        cout << arr[i] << " ";
    return 0;
}

File: Code/py/selection_sort.py
arr = []
length = int(input("Enter Length: "))

for i in range(length):
    arr.append(int(input("Enter Number: ")))

for i in range(length):
    for j in range(i + 1, length):
        if arr[i] > arr[j]:
            arr[i], arr[j] = arr[j], arr[i]
print(arr)